src/apps/testapps/testVertex.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file
+ * @brief tests H3 vertex functions.
+ *
+ *  usage: `testVertex`
+ */
+
+#include "test.h"
+#include "utility.h"
+#include "vertex.h"
+
+SUITE(Vertex) {
+    TEST(vertexNumForDirection_hex) {
+        H3Index origin = 0x823d6ffffffffff;
```

Comment:
Does this need the `L` suffix?

Reply:
Apparently not? We don't seem to be consistent, and there are a number of indexes in tests and benchmarks that don't include the `L` suffix and work fine. Is there a reason we thought it was required?